test/nms-openwire-test.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
missing  package id, e.g.   <PackageId>Apache.NMS.OpenWire.Test</PackageId>

Reply:
Not sure if there is need to publish tests as nuget package at all.

Reply:
true, and thats down to if we want to publish it to nuget at part of release, but still good to have uniform setup across them all, then its just down to should we wish to publish or not.